x/blob/README.md

```diff
@@ -2,44 +2,141 @@
 
 ## Abstract
 
-The `x/blob` module enables users to pay for arbitrary data to be published to the Celestia blockchain. Users create a single `BlobTx` that is composed of:
+The `x/blob` module enables users to pay for arbitrary data to be published to
+the Celestia blockchain. Users create a single `BlobTx` that is composed of:
 
-1. Multiple `Blob`s (Binary Large OBjects): the data they wish to publish. A single `Blob` is composed of:
+1. Multiple `Blob`s (Binary Large OBjects): the data they wish to publish. A
+   single `Blob` is composed of:
     1. `NamespaceId  []byte`: the namespace this blob should be published to.
     1. `Data         []byte`: the data to be published.
-    1. `ShareVersion uint32`: the version of the share format used to encode this blob into a share.
+    1. `ShareVersion uint32`: the version of the share format used to encode
+       this blob into a share.
 1. A single `sdk.Tx` which is composed of:
```

Comment:
Can we first split this into:
```
1. A single `sdk.Tx` which is composed of:
   1. One and only one `MsgPayForBlob`
   2. Any other sdk messages.
```

And then in the PFB part explain the subcomponents

Reply:
Ah no wait ignore, I realise we don't support multiple other sdk messages in a `BlobTx`